docs/API/V1/list.md

```diff
@@ -70,19 +70,22 @@ and only `public` access datasets will be selected.
 The return is a serialized JSON object with information about the selected
 datasets.
 
+## Resource access
+
+* Only `<dataset>` resources selected by the filter to which the authenticated
+user has READ access will be returned.
+
+See [Access model](../access_model.md)
+
 ## Response status
 
 `401`   **UNAUTHORIZED** \
-The client did not provide an authentication token, and there is no public
-dataset with the name `<dataset>`.
+The client did not provide an authentication token but asked to filter datasets
+by `owner` or `access=private`.
```

Comment:
I'm confused:  if I'm not logged in, I cannot request public datasets which are owned by a particular user?

Reply:
We went back and forth on this a long time ago; but if we want unauthenticated clients to be able to look up usernames, we'd need to change the validator and the messaging.